### PR TITLE
M62 release notes

### PR DESCRIPTION
**Pull request by infin8x**

### CODE REVIEW

File: themes/default/content/blog/pulumi-release-notes-m62/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+This release also includes a new tool for migrating existing CloudFormation templates into Pulumi programs in your favorite language, powered by the new AWS Native provider and AWS Cloud Control API, as well as the ability to deploy any 3rd party resources in the CloudFormation Registry, including resources from Atlassian, Datadog, Densify, Dynatrace, Fortinet, New Relic, and Spot by NetApp.
+
+Learn more in the [Announcing AWS Native]({{< relref "/blog/announcing-aws-native" >}}) blog post
+
+### Using AWS Graviton2 processors in AWS Lambda functions
+
+Last week, AWS launched the ability to use Graviton2 processors with AWS Lambda functions. Using Graviton2 processors gives you access to the improved price performance of that architecture. Pulumi's AWS Classic provider shipped with same-day support for this new functionality.
+
+Learn more in the [blog post]({{< relref "/blog/aws-lambda-functions-powered-by-graviton2" >}})
```
Comment (susanev) [inline themes/default/content/blog/pulumi-release-notes-m62/index.md:38]:
```suggestion
Learn more in the [AWS Lambda functions blog post]({{< relref "/blog/aws-lambda-functions-powered-by-graviton2" >}})
```

File: themes/default/content/blog/pulumi-release-notes-m62/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+### New resources in the Azure Native provider
+
+We shipped new versions of the Azure Native provider (1.29.0 through 1.34.0) that collectively added 14 new resources across services like Azure SQL, Azure API Management, and more that you can now manage with the Azure Native provider.
+
+[See the full list](https://github.com/pulumi/pulumi-azure-native/blob/master/CHANGELOG.md#1340-2021-09-30)
```
Comment (susanev) [inline themes/default/content/blog/pulumi-release-notes-m62/index.md:44]: would rec linking to the full list in the 14 new resources mention above

File: themes/default/content/blog/pulumi-release-notes-m62/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Pulumi CLI and core technologies
+
+In this milestone, we shipped Pulumi versions [3.13.0](https://github.com/pulumi/pulumi/releases/tag/v3.13.0) through [3.13.2](https://github.com/pulumi/pulumi/releases/tag/v3.13.2). The full list of changes in each version is available [in the changelog](https://github.com/pulumi/pulumi/releases); read on to learn about some of the biggest changes.
```
Comment (susanev) [inline themes/default/content/blog/pulumi-release-notes-m62/index.md:48]:
```suggestion
In this milestone, we shipped Pulumi versions [3.13.0](https://github.com/pulumi/pulumi/releases/tag/v3.13.0) through [3.13.2](https://github.com/pulumi/pulumi/releases/tag/v3.13.2). The full list of changes in each version is available in the [changelog](https://github.com/pulumi/pulumi/releases); read on to learn about some of the biggest changes.
```

**Review by susanev — COMMENTED** [review]
link text really needs to be consumable outside of the context that it is within

**Review by emiliza — APPROVED** [review]

**Pull request merged**